### ISSUE TITLE
Timestamps

### ISSUE THREAD

**Issue by weinholt** [lassik/scheme-release-archive#1]
Hello @lassik! What do you think about preserving timestamps on the archives in this repo? Git doesn't preserve them unfortunately, but I suspect that you might have the majority of them locally if you've used wget or other normal tools to download them.

I don't know if there are any tools around to maintain this info automatically, but doing it by hand would not be workable.

**Comment by lassik** [comment]
Good idea. Unfortunately, I don't have good timestamps for most of the archives even locally.

Here are the archives for which my local copies have a date in the 1990s decade:

```
1992 RABBIT.gz
1992 fools.1.3.2.tar.gz
1992 pixiescheme.cpt.hqx
1992 xscheme-0.28.tar.gz
1993 Similix5.0.tar.gz
1994 pseudo212.tar.gz
1994 scm4e1.tar.gz
1994 xscheme0.2.tar.gz
1995 elk-3.0.tar.gz
1995 hobbit4d.tar.gz
1996 siod-3.2.tar.gz
1997 STk-3.1.1.tar.gz
1997 gps10bin.zip
1997 gps10src.zip
1997 mfc40dll.zip
1999 STk-4.0.0.tar.gz
1999 STk-4.0.1.tar.gz
```

**Comment by lassik** [comment]
There is some kind of `.git_meta` file that can track file modification times ([example](https://github.com/egallesio/STklos/blob/master/.git_meta)).

**Comment by lassik** [comment]
There's also the question of whether the original file had a sensible timestamp to begin with. To figure out when a release was made, it's probably best to look for a readme in the archive, or to parse dates from the file listing of each zip/tar archive. I've kept the zip/tar files bit-identical to the originals so checksums can be compared for veracity.

**Comment by lassik** [comment]
Here's a simple utility to list the dates of all the files inside an archive: https://github.com/lassik/archive-dater
